developer-docs-site/docs/tutorials/validator-node/using-docker.md

```diff
@@ -64,7 +64,11 @@ Docker has only been tested on Linux, Windows, and Intel macOS. If you are on M1
         --full-node-host fn.bot.aptosdev.com:6182
     ```
 
-    This will create a YAML file in your working directory with your username, e.g. `aptosbot.yml`, it should looks like:
+    This will create a YAML file in your working directory with your username, e.g. `aptosbot.yml`
+    
+    `cat aptosbot.yml`
+    
+    it should looks like:
```

Comment:
this seems ... unnecessary?

Reply:
Seems, but maybe it will help someone without asking on Discord.